disk_areas.md

```diff
@@ -0,0 +1,78 @@
+
+Abel Disk areas that are relevant for your work at the Veterinary Institute
+
+# Summary :
+
+|directory|path                |what      |characteristics|
+|:--------|:-------------------|:---------|:--------------|
+|`$HOME`|`usit/abel/u1/username`|"home-home" area|500GB - login area- NOT FOR Data Processing -backed-up|
+|**work**|`/work/projects/nn9305k`|common resources (eg. softwares, databases)|10 TB - faster disc, closer to nodes connection|
+|**project**|`/projects/nn9305k`|Project data area (backed-up, raw data, results)|30 TB - slower disc|
+|`$SCRATCH`|`/work/jobs/JOBID.d`|computing nodes: **FOR PROCESSING DATA = RUNNING JOBS**|fastest|
+|`$USERWORK`|`/work/users/username`|sharing data among different jobs = **staging**|do not backed-up - removed after 45 days|
+
+1. **$HOME**: "home-home" directory (on login nodes): 500GB - backed up regularly: slow and costly.
+  - If you need to store files that do not require backup: put them in a `nobackup` directory
+  - recommended for: software, job configurations, job preparation, debugging ...
+  - DO NOT USE to run jobs => long jobs should be run on $SCRATCH (submitted before hand to the queue system **SLURM**)
+
+QQQ: -> **login nodes** NB: small tests are tolerated (processes taking more than 30 min are automatically killed) automatically <=> so in home-home
+
+**work area**: `/work/projects/nn9305k` will be used to contain:
+- softwares and libraries
+- public reference data, template files, adapters files ...
+
+2. **project area**: `/projects/nn9305k` will be used to contain:
+ - project data, including raw sequencing files (zipped/tared:compressed and decompressed)
+ - your `project/home/username`
+ - data analyses that have been performed
+ - ... this is where the work will NOW be done.
+
+QQQ - [ ]-> **so our(vetinst users) /work/projects/nn9305k/home/username (incl. all subfolder should be moved here)** -> `project/home/username`
+
+3. **$SCRATCH**: when a job is started, a temporary directory on $SCRATCH is automatically created. It is also automatically deleted when the job finishes. This allows the jobs to run faster without interfering with the work of others. You can access $SCRATCH for status monitoring of running jobs: with `/work/jobs/JOBID.d` where JOBID.d is the ID that has been provided when you submitted the job in the queue system SLURM.
+
+4. **$USERWORK** If files are needed for more than one job, they are **staged** in here. Files are automatically deleted after 45 days, and there is no backup. Access: `/work/users/username`
```

Comment:
you should put your files there, and run analysis from that area, either via slurm script or other mechanisms.

Reply:
other mechanisms ? qlogin? command line? ...maybe will also find out later on

Reply:
added to text